ofborg/src/tasks/evaluate.rs

```diff
@@ -334,6 +334,14 @@ impl<'a, E: stats::SysEvents + 'static> OneEval<'a, E> {
             None => String::from("master"),
         };
 
+        if target_branch.starts_with("nixos-") || target_branch.starts_with("nixpkgs-") {
+            overall_status
+                .set_with_description("Improper target branch", hubcaps::statuses::State::Error)?;
```

Comment:
@LnL7 Something like this?

```suggestion
                .set_with_description("The branch you have targeted is a read-only mirror for channels. Please target release-* or master.", hubcaps::statuses::State::Error)?;
```

Reply:
Yeah, something like that is a bit more actionable.